source/dyaml/node.d

```diff
@@ -2446,12 +2428,30 @@ struct Node
         // Safe wrapper for getting a value out of the variant.
         inout(T) getValue(T)() @trusted inout
         {
-            return value_.get!T;
+            alias RType = typeof(return);
+            return value_.tryMatch!((RType r) => r);
         }
         // Safe wrapper for coercing a value out of the variant.
         inout(T) coerceValue(T)() @trusted inout
         {
-            return (cast(Value)value_).coerce!T;
+            alias RType = typeof(return);
```

Comment:
Would also like this one to be `@safe` or documented

Reply:
Likewise, I'd recommend to delay opening this can of worm for another PR>

Reply:
Looks like this can be made `@safe` if we add a `toString` to `Node`. I'll look at it tomorrow.

Reply:
Sounds good to me.